src/modules/authentication/api/person-time-limit-info.reponse.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { TimeLimitOccasion } from '../../person/domain/time-limit-occasion.enums.js';
+
+export class PersonTimeLimitInfoResponse {
+    @ApiProperty({ nullable: true })
+    public occasion: TimeLimitOccasion;
+
+    @ApiProperty({ nullable: true })
+    public deadline: string;
+
+    public constructor(occasion: TimeLimitOccasion, deadline: Date) {
+        this.occasion = occasion;
+        this.deadline = deadline?.toISOString();
+    }
```

Comment:
Are these nullable? They can't be undefined in the constructor (and the nullish operator in line 13 is unused)